static/app/views/replays/detail/domMutations/index.tsx

```diff
@@ -22,60 +20,67 @@ function DomMutations({replay}: Props) {
 
   const startTimestamp = replay.getEvent().startTimestamp;
 
+  if (isLoading) {
+    return null;
```

Comment:
is this sufficient?

Reply:
Yeah the parent tab section already handles the loading state so that is there 'just in case'. It doesn't ever render when I was testing it